eslint.config.js

```diff
@@ -1,9 +1,43 @@
-import antfu from '@antfu/eslint-config'
+// @ts-check
+import { createConfigForNuxt } from '@nuxt/eslint-config/flat'
 
-export default antfu({
-  rules: {
-    'no-use-before-define': 'off',
```

Comment:
### Note

This rule is an additional rule specific to `@antfu/eslint-config`.

https://github.com/antfu/eslint-config/blob/main/src/configs/javascript.ts#L154